templates/item/item-ability-sheet.html

```diff
@@ -24,6 +24,7 @@ <h1 class="charname"><input name="name" type="text" value="{{item.name}}" placeh
                 <div class="flexcol flexrow">
                     <select name="data.defaultCharacteristicForAbility" data-type="String" class="default-characteristic">
                         {{#select data.defaultCharacteristicForAbility}}
+                        <option value="">{{localize "arm5e.sheet.n-a"}}</option>
```

Comment:
I see a potential problem here, since the default value is empty but become "N/A" the first time you open the ability, what happen if you try to roll in one case or the other?